Fabric.Identity.API/Persistence/SqlServer/Stores/SqlServerUserStore.cs

```diff
@@ -90,19 +81,21 @@ public void UpdateUser(User user)
 
             user.ToEntity(existingUser);
 
+            foreach (var claim in user.Claims)
+            {
+                var userClaim = existingUser.Claims.FirstOrDefault(c => c.Type == claim.Type);
+                if (userClaim != null)
+                {
+                    userClaim.Value = claim.Value;
+                }
+                else
+                {
+                    existingUser.Claims.Add(new UserClaim{Type = claim.Type, Value = claim.Value});
+                }
+            }
+
```

Comment:
you could move this line into the ToEntity extension method that accepts an existing entity as a parameter